english/impossible-girl/impossible.md

```diff
@@ -3,3 +3,39 @@ You return to the marshmallow realm and begin to search for Clara.
 Suddenly, your blood runs cold as you hear a familiar mechanical voice greet you.  "Exterminate!"
 
 You turn around and see a Dalek
+
+Or is it?
+
+As your eyes adjust to the gloom, you suddenly realize that it is just a child wearing a topside down rubbish bin,
+and holding a toilet plunger.  He thrusts the plunger in your general direction, screaming "EXTERMINATE!" once more.
+What a strange child, you think.  Surely he realizes the danger of impersonation of the deadliest entities ever to
+exist...
+
+Just then, you hear a noise.  It is a dim sound, just on the edge of your consciousness.  You could almost
+think you were imagining it.
+
+thud... thud... thud...
+
+You hold your breath in anticipation... Nothing.  Just the wind.  Just the darkness.  Just the...
+
+"EXTERMINATE!," screams the boy again, lunging at you with such force that you are certain to get a bruise.
+
+"EXTERMI...," he starts.
+
+THUD... THUD... THUD...
+
+He freezes in place, his eyes growing wide.
+
+THUD... THUD... THUD...
+
+He whirls before the echoes die, holding the plunger aloft as if to repel the...well, whatever it is.  And then he
+is gone.  Without a "by your leave" or a "ta ta," he runs.  He runs as if his life depended upon it.  And you watch
+as the darkness englulfs him.
+
+THUD... THUD... THUD...
+
+Do you:
+
+[Ignore the noise and taste a bit of the marshmallow wall you just noticed to your right.](../eating-walls/eating-marshmallows.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.